src/com/etendoerp/openapi/model/window/WindowSettingEndpoint.java

```diff
@@ -24,6 +26,15 @@ public class WindowSettingEndpoint implements OpenAPIEndpoint {
   private static final String WINDOW_SETTINGS_ACTION = "org.openbravo.client.application.WindowSettingsActionHandler";
   private static final String FORM_INIT_ACTION = "org.openbravo.client.application.window.FormInitializationComponent";
   private static final String BASE_PATH = "/etendo/org.openbravo.client.kernel?_action=";
+  private static final List<String> tags = List.of("Window Settings");
+
+  @Override
+  public boolean isValid(String tag) {
```

Comment:
This method is in three different java files. Wouldn't it be better for this to be in a "Util" file?

Reply:
It uses a private propertie so its not possible. Additionally this method can be different for each class that implements the Interface